R/zchunk_L261.Cstorage.R

```diff
@@ -1,20 +1,34 @@
 #' module_energy_L261.Cstorage
 #'
-#' Briefly describe what this chunk does.
+#' Calculates carbon storage resource supply curves, shareweights, technology coefficients and costs, and other carbon storage information
 #'
 #' @param command API command to execute
 #' @param ... other optional parameters, depending on command
 #' @return Depends on \code{command}: either a vector of required inputs,
 #' a vector of output names, or (if \code{command} is "MAKE") all
 #' the generated outputs: \code{L261.DepRsrc}, \code{L261.UnlimitRsrc}, \code{L261.DepRsrcCurves_C}, \code{L261.SectorLogitTables[[ curr_table ]]$data}, \code{L261.Supplysector_C}, \code{L261.SubsectorLogitTables[[ curr_table ]]$data}, \code{L261.SubsectorLogit_C}, \code{L261.SubsectorShrwtFllt_C}, \code{L261.StubTech_C}, \code{L261.GlobalTechCoef_C}, \code{L261.GlobalTechCost_C}, \code{L261.GlobalTechShrwt_C}, \code{L261.GlobalTechCost_C_High}, \code{L261.GlobalTechShrwt_C_nooffshore}, \code{L261.DepRsrcCurves_C_high}, \code{L261.DepRsrcCurves_C_low}, \code{L261.DepRsrcCurves_C_lowest}. The corresponding file in the
 #' original data system was \code{L261.Cstorage.R} (energy level2).
-#' @details Describe in detail what this chunk does.
+#' @details The below fifteen tables pertaining to carbon storage properties are generated:
+#'     Carbon storage information
```

Comment:
Please put these into an easy-to-read list: https://stackoverflow.com/questions/9267584/when-documenting-in-roxygen-how-do-i-make-an-itemized-list-in-details